ironfish-cli/src/commands/wallet/chainport/send.ts

```diff
@@ -0,0 +1,346 @@
+    const { flags } = await this.parse(BridgeCommand)
+    let from = flags.account?.trim()
+    let to = flags.to?.trim()
+
+    const networkId = (await client.chain.getNetworkInfo()).content.networkId
+
+    if (networkId !== TESTNET.id) {
+      CliUx.ux.error('This command is only available on testnet')
+    }
+
+    if (!from) {
+      const response = await client.wallet.getDefaultAccount()
+
+      if (!response.content.account) {
+        this.error(
+          `No account is currently active.
+           Use ironfish wallet:create <name> to first create an account`,
+        )
+      }
+
+      from = response.content.account.name
+    }
+
+    if (!to) {
+      to = await CliUx.ux.prompt('Enter the public address of the recipient', {
+        required: true,
+      })
+    }
+
+    if (validator.isAddress(to) === false) {
+      this.error('Invalid to ethereum address')
+    }
+
+    const assetId = Asset.nativeId().toString('hex')
+    const assetData = (
+      await client.wallet.getAsset({
+        account: from,
+        id: assetId,
+      })
+    ).content
+
+    const selectedNetwork = await this.selectNetwork()
+
+    let amount
+    if (flags.amount) {
+      const [parsedAmount, error] = CurrencyUtils.tryMajorToMinor(
+        flags.amount,
+        assetId,
+        assetData?.verification,
+      )
+
+      if (error) {
+        this.error(`${error.message}`)
+      }
+
+      amount = parsedAmount
+    }
+
+    if (amount === null || amount === undefined) {
+      amount = await promptCurrency({
+        client: client,
+        required: true,
+        text: 'Enter the amount in the major denomination',
+        minimum: 1n,
+        logger: this.logger,
+        assetId: assetId,
+        assetVerification: assetData.verification,
+        balance: {
+          account: from,
+        },
+      })
+    }
+
+    const { outputs, raw }: { outputs: BridgeTransactionDetails; raw: RawTransaction } =
+      await this.constructBridgeTransaction(amount, assetId, selectedNetwork, to, from, client)
+
+    this.bridgeSummary(outputs, raw, from, to, assetId, assetData)
+
+    const confirmed = await CliUx.ux.confirm('Do you confirm (Y/N)?')
+    if (!confirmed) {
+      this.error('Transaction aborted.')
+    }
+
+    const postTransaction = await client.wallet.postTransaction({
+      transaction: RawTransactionSerde.serialize(raw).toString('hex'),
+      account: from,
+    })
+
+    const bytes = Buffer.from(postTransaction.content.transaction, 'hex')
+    const transaction = new Transaction(bytes)
+
+    if (postTransaction.content.accepted === false) {
+      this.warn(
+        `Transaction '${transaction.hash().toString('hex')}' was not accepted into the mempool`,
+      )
+    }
+
+    const transactionUrl = getExplorer(networkId)?.getTransactionUrl(
+      transaction.hash().toString('hex'),
+    )
+
+    if (transactionUrl) {
+      this.log(`\nIf the transaction is mined, it will appear here: ${transactionUrl}`)
```

Comment:
transaction will show up in block explorer even if it isn't mined. Can remove tha caveat.